homeassistant/components/pjlink/media_player.py

```diff
@@ -83,12 +84,22 @@ def __init__(self, host, port, name, encoding, password):
         self._encoding = encoding
         self._source_name_mapping = {}
 
-        self._attr_name = name
+        self._attr_available = False
         self._attr_is_volume_muted = False
-        self._attr_state = MediaPlayerState.OFF
+        self._attr_manufacturer = None
+        self._attr_model = None
+        self._attr_name = name
         self._attr_source = None
         self._attr_source_list = []
-        self._attr_available = False
+        self._attr_state = MediaPlayerState.OFF
+        self._attr_unique_id = host
```

Comment:
If anyone has any better ideas on how to set the unique ID I would love to hear it. :grin: